### ISSUE TITLE
Recommended values for training models

### ISSUE THREAD

**Issue by hannes-michael** [conan7882/fast-style-transfer-tensorflow#1]
What are recommended values for training models (batch size / learning rate)?
```
python3 fast.py --train \
   --batch BATCH_SIZE \
   --lr LEARNING_RATE \
   --styleim FILE_NAME_OF_STYPE_IMAGE \
```

And minor issue: FILE_NAME_OF_**STYPE**_IMAGE 
I guess it is supposed to be "FILE_NAME_OF_STYLE_IMAGE"
